docs/build/guides/conventions/workspace.mdx

```diff
@@ -0,0 +1,357 @@
+
+```
+stellar contract build
+```
+
+As seen previously, target wasm file is located in
+
+```
+$PROJECT_ROOT/target/wasm32-unknown-unknown/release/
+```
+
+In order to reference this wasm file from `contracts/main_contract/src/lib.rs`, we need to first construct the path to the project root: `../..` (relative to second contract's Cargo.toml file location) followed by path to the wasm file:
+
+```
+../../../target/wasm32-unknown-unknown/release/add_contract.wasm
+```
+
+It can now be used in the rust code:
+
+```rust title="contracts/main_contract/src/lib.rs"
+#![no_std]
+
+use soroban_sdk::{contract, contractimpl, Address, Env};
+
+mod contract_add {
+    soroban_sdk::contractimport!(
+        file = "../../../target/wasm32-unknown-unknown/release/add_contract.wasm"
+    );
+}
+
+#[contract]
+pub struct ContractMain;
+
+#[contractimpl]
+impl ContractMain {
+    pub fn add_with(env: Env, contract: Address, x: u32, y: u32) -> u32 {
+        let client = contract_add::Client::new(&env, &contract);
+        client.add(&x, &y)
+    }
+}
+
+mod test;
+```
+
+Here, main contract will invoke `ContractAdd`'s `add` function to calculate the sum of 2 numbers. It's a good idea to update tests for our main contract as well:
+
+```rust title="contracts/main_contract/src/test.rs"
+#![cfg(test)]
+
+use crate::{contract_add, ContractMain, ContractMainClient};
+use soroban_sdk::Env;
+
+#[test]
+fn test_adding_cross_contract() {
+    let env = Env::default();
+
+    // Register add contract using the imported WASM.
+    let contract_add_id = env.register_contract_wasm(None, contract_add::WASM);
+
+    // Register main contract defined in this crate.
+    let contract_main_id = env.register_contract(None, ContractMain);
+
+    // Create a client for calling main contract.
+    let client = ContractMainClient::new(&env, &contract_main_id);
+
+    // Invoke main contract via its client. Main contract will invoke add contract.
+    let sum = client.add_with(&contract_add_id, &5, &7);
+    assert_eq!(sum, 12);
+}
+```
+
+Contracts can now be re-complied running the following command from the project root:
+
+```
+stellar contract build
+```
+
+And check that the test is working correctly, running tests in `contracts/main_contract`:
+
+```
+cargo test
+
+running 1 test
+test test::test_adding_cross_contract ... ok
+
+test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.01s
+```
+
+Finally, let's deploy this contracts and call our main contract using cli. If you haven't already, set up an account (alice) first
+
+```bash
+stellar keys generate alice --fund --network testnet
+STELLAR_SOURCE_ACCOUNT=alice
```

Comment:
The account can be set in a way that is platform agnostic using:
```suggestion
stellar keys use alice
```

Reply:
Good catch, I believe `stellar keys use` wasn't live yet when I wrote this guide